package.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "rest-client-app",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev --turbopack",
+    "build": "next build --turbopack",
+    "start": "next start",
+    "lint": "eslint"
```

This is too generic. Please use either:

```suggestion
    "lint": "next lint"
```

or 
```suggestion
    "lint": "eslint . --ext .ts,.tsx"
```

---

sorry it was generated default by `npx create-next-app@latest .`. 
I'll change it next steps to  "lint": "eslint . "